contrib/mod_sftp_sql.c

```diff
@@ -50,7 +50,7 @@ struct sqlstore_data {
 
 static const char *sqlstore_user = NULL;
 
-static const char *trace_channel = "ssh2";
+static const char *trace_channel = "sftp.sql";
```

Comment:
Note that I created a separate trace channel for this module.  I've found that, as I develop more modules, having focused trace channels whose names align more closely with their module makes the logging easier to follow.